fcrepo-kernel-api/src/main/java/org/fcrepo/kernel/api/services/VersionService.java

```diff
@@ -47,13 +47,13 @@
     /**
      * To format a datetime for use as a Memento path.
      */
-    public static final DateTimeFormatter MEMENTO_LABEL_FORMATTER = DateTimeFormatter.ofPattern("yyyyMMddHHmmss")
+    DateTimeFormatter MEMENTO_LABEL_FORMATTER = DateTimeFormatter.ofPattern("yyyyMMddHHmmss")
```

Comment:
Does java automatically assume the `static final` for variables in interfaces? I didn't realize that.

Reply:
Yes. https://www.javahelps.com/2015/02/introduction-to-interface-with-java-8.html